master/reference/calicoctl/resources/bgpconfig.md

```diff
@@ -0,0 +1,47 @@
+---
+title: BGP Configuration Resource (BGPConfiguration)
+---
+
+A BGP configuration resource (BGPConfiguration) represents BGP specific configuration options for the cluster.
+
+For `calicoctl` commands that specify a resource type on the CLI, the following
+aliases are supported (all case insensitive): `bgpconfiguration`, `bgpconfig`, `bgpconfigurations`, `bgpconfigs`.
```

Comment:
Can you give an example?

Reply:
The examples will be in `Reference>calicoctl>commands` section

Reply:
OK, that will be great. Then maybe we can just link them there for the extra info.

Reply:
done